ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.schema.yml

```diff
@@ -6819,6 +7077,68 @@ keys:
                 description: 'Text comment added to queue.
 
                   '
+              random_detect:
+                type: dict
+                keys:
+                  ecn:
+                    description: Explicit Congestion Notification
+                    type: dict
+                    keys:
+                      count:
+                        type: bool
+                        description: 'Enable counter for random-detect ECNs
+
+                          '
+                      threshold:
+                        type: dict
+                        keys:
+                          units:
+                            required: true
+                            type: str
+                            valid_values:
+                            - segments
+                            - bytes
+                            - kbytes
+                            - mbytes
+                            description: 'Unit to be used for the threshold values.
+
+                              '
+                          min:
+                            type: int
+                            required: true
+                            min: 1
+                            convert_types:
+                            - str
+                            description: 'Random-detect ECN minimum-threshold.
+
+                              '
+                          max:
+                            type: int
+                            required: true
+                            min: 1
+                            convert_types:
+                            - str
+                            description: 'Random-detect ECN maximum-threshold.
+
+                              '
+                          max_probability:
+                            type: int
+                            min: 1
+                            max: 100
+                            convert_types:
+                            - str
+                            description: 'Random-detect ECN maximum mark probability.
+
+                              '
+                          weight:
+                            type: int
+                            min: 1
```

Comment:
the min for the weight could be between 0 and 15, see cli:

ld002(config-s-test-qos-profile-NET-INFRA-txq-4)# random-detect ecn minimum-threshold 256 kbytes maximum-threshold 512 kbytes weight ?
  <0-15>  Value of ECN weight